### ISSUE TITLE
change installation to install with apt 

### ISSUE THREAD

**Issue by darkkLUCIFER** [lnxpy/hey#18]
* Hey version:
* Python version: 3.12.3
* Operating System: Ubuntu 24.04 LTS

### Suggest
make the installation more easy with apt 

@lnxpy 

### Description

i tried to install it with pip in my whole system but it dont allow to install without using an environment as simple 
and needs to use some commands 

### What I Did

i used the below commands to install
sudo pip install -U hey-mindsdb --break-system-packages

and after using this i raise some errors about package that was installed by the python named typing-extensions and forced me to use the below command 
sudo pip install -U hey-mindsdb --break-system-packages --ignore-installed typing-extensions

**Comment by amirsoroush** [comment]
> and after using this i raise some errors about package that...

What errors exactly? provide the traceback.

Did you update the `pip` before installation?

Also, to install system-wide Python apps you may take a look at [pipx](https://github.com/pypa/pipx) instead.

**Comment by lnxpy** [comment]
@amirsoroush Good point, but not all the Linux users are familiar with Python and its package management. They are quite experienced at using their OS-specific package managers. Also–they wouldn't need to install `pipx` to have Hey installed on their machines.

**Comment by lnxpy** [comment]
What do you mean by "switches" here, @amirsoroush?!

> Apparently by using those switches he is familiar with the package manager..

Well, the user might not be familiar with Python tools like `pipx` or even `pip`. Package managers like `apt` and `pacman` are installed by default on the distros. No matter the user's stack or expertise, he knows how to work with them for sure. Additionally, by using them, he goes straight to the point and installs the actual package other than messing around with installing the `pipx` and then going for the package installation. That's my point!

**Comment by lnxpy** [comment]
@amirsoroush 😅🤙

**Comment by sepehrrasooli** [comment]
Is this issue still active @lnxpy ?

**Comment by lnxpy** [comment]
Yes, it is. We have some refactoring steps to do before this one. @sepehrrasooli
